### ISSUE TITLE
Precision issue (Maybe)

### ISSUE THREAD

**Issue by Hoodgail** [gkjohnson/three-mesh-bvh#520]
**Describe the bug**
Hi, so i'm getting precision between my game's server an the client, i've went through the whole code thinking that it was me doing something wrong, i tried everything i could, but never found a solusion, so i came to a conclusion that it might be this library, but i'm still not sure.
I wanted to ask if there is a precision issue in bvh.shapecase, intersectsTriangle where it might not call intersectsTriangle with the same amount of triangles sometimes.

**To Reproduce**
For example, at https://github.com/gkjohnson/three-mesh-bvh/blob/master/example/characterMovement.js#L305
Using the same delta, same initial velocity & position, that function would be called on the browser, and on nodejs, the same amount of time... lets say for about 100k frames, the positions should be different at some point which it shoulnt which is my problem, been trying to find the issue for months, its making me go crazy.

 
**Screenshots**
![image](https://user-images.githubusercontent.com/45743294/231061306-c8f52969-b3c9-482b-9cf8-26128eb24f9e.png)
First number is the packet id, second is the position distance from the server and the client (Has nothing to do with latency, i gave each inputs an Initial sequence number (ISN) to cache previous values since the server was usually 6 frames behind.

**Video**
https://pub-f1c26332ec954985aad785db2167ca2f.r2.dev/2023-04-11_01-25-25.mp4

Look at the console tab, getting so many mismatch, but idk, i might be the one doing something wrong somewhere & can't find where.

**Platform:**
 - Device: [Desktop]
 - OS: [Windows]
 - Browser: [Chrome, NodeJS]
 - Three.js version: [r150]
 - Library version: [v0.5.21]

**Comment by gkjohnson** [comment]
If you're passing the exact same values into the exact same BVH I would expect them to be the same. If you're seeing otherwise then I'd need to see a live example of a case where this is happening to understand what might be going on.

**Comment by Hoodgail** [comment]
Sorry for bothering you.

I have identified and resolved an issue that was occurring. I discovered that the root cause of the problem was related to the communication between the server and client. Specifically, I was transmitting positional data from the server to the client at 1-second intervals. While I believed this approach to be effective, it ultimately resulted in synchronization problems due to the server lagging behind the client. As a result, the client was consistently falling behind every second, leading to a loss of synchronization and mis-precision.
